cc/scripts/scripts.js

```diff
@@ -13,23 +13,24 @@
 import { setLibs } from './utils.js';
 
 // Add project-wide style path here.
-const STYLES = '';
+const STYLES = '/cc/styles/styles.css';
 
 // Use '/libs' if your live site maps '/libs' to milo's origin.
 const LIBS = 'https://milo.adobe.com/libs';
 
 // Add any config options.
 const CONFIG = {
-  // codeRoot: '',
-  // contentRoot: '',
-  // imsClientId: 'college',
-  // geoRouting: 'off',
```

Comment:
nit - please remove this comment . You don't need to send those bytes to the client anymore :) 
`// geoRouting: 'off',`

Reply:
The line with this comment is actually marked deleted and there is nothing that needs to be removed there.
Please correct me if I am wrong.